MIP0/mip0.md

```diff
@@ -1,3 +1,7 @@
+---
+tags: MIP0
```

Comment:
Is this part of HackMD's stuff?

Reply:
Ah, yes. Forgot to remove the YAML. Sorry